controllers/users.js

```diff
@@ -21,14 +21,14 @@ exports.read = function(req, res) {
 		if (!user) {
 			return res.status(404).send("This user does not exist.");
 		} else {
-      var sanitizedUser = {
-        name: user.name,
-        meetings: user.meetings,
-        contacts: user.contacts
-      };
+//      var sanitizedUser = {
+//        name: user.name,
+//        meetings: user.meetings,
+//        contacts: user.contacts
+//      };
 
 
-			res.json(sanitizedUser);
+			res.json(user);
```

Comment:
This function is looking for a user ID that any user can submit in the req.params. That means I can look up someone else's profile information and get their password, because it's only using the req parameters to find them. All I have to do is make a get request to /api/users/:otherperson and it gives me their password.

If we're going to return the whole user object, then we should not make this functionality accessible through req.params, but only req.user.id (as it only returns that user's information who is logged in and has a session).

If we're going to let the controller get all user objects from params, then we need to return the sanitized user, with no sensitive information.

So, either: req.params.user_id returns sanitized information, or req.user.id (only the user who is currently logged in) returns their whole user object.